rest-api/tools/export_tables.py

```diff
@@ -0,0 +1,76 @@
+"""Uses Cloud SQL export API to export the contents of one more tables to CSV files.
+"""
+
+import logging
+import sys
+
+from time import sleep
+from main_util import get_parser, configure_logging
+from googleapiclient import discovery
+from oauth2client.service_account import ServiceAccountCredentials
+
+_SCOPE = "https://www.googleapis.com/auth/cloud-platform"
+_INSTANCE = "rdrmaindb"
+
+def main(args):
+  credentials = ServiceAccountCredentials.from_json_keyfile_name(args.creds_file, [_SCOPE])
+  service = discovery.build('sqladmin', 'v1beta4', credentials=credentials)
+  table_names = args.tables.split(',')
+  for table in table_names:
+    uri = args.output_path + '/%s.csv' % table
+    logging.info("Exporting %s.%s to %s..." % (args.database, table, uri))
+    request_body = {
+      "exportContext": {
+        "kind": "sql#exportContext",
+        "fileType": "CSV",
+        "uri": uri,
+        "databases": [
+          args.database
+        ],
+        "csvExportOptions": {
+          "selectQuery": "select * from %s" % table
```

Comment:
We should either use a SQL escaping library or make it very clear the table name is going into SQL raw.

Reply:
Done.